docs/src/app/components/pages/components/auto-complete.jsx

```diff
@@ -21,56 +21,62 @@ class AutoCompletePage extends React.Component {
 
     return (
       <ComponentDoc
-        name="Auto Complete"
-        componentInfo={[{
-          name: 'Auto Complete',
-          infoArray: [],
-        }]}>
+        name = "Auto Complete"
+        componentInfo = {[
+          {
+            name: 'Auto Complete',
+            infoArray: [],
+          },
+        ]}>
 
         <br/>
 
-        <CodeExample code={Code}>
+        <CodeExample code = {Code}>
```

Comment:
Could you remove the spaces between the `=` to stay coherent with the code base?
Thanks